toolkit/templates/forms/selection-buttons.html

```diff
@@ -24,9 +26,9 @@
       </label>
     {% else %}
       {% for option in options %}
-        <label class="selection-button" for="{{ name }}-{{ loop.index }}">
+        <label class="selection-button{% if inline is defined %} selection-button-inline{% endif %}" for="{{ name }}-{{ loop.index }}">
```

Comment:
`if inline` would be better here. As is, `inline: False` does the opposite of what you'd expect.